guides/v2.1/cloud/project/new-relic.md

```diff
@@ -1,44 +1,34 @@
 ---
 group: cloud-guide
-title: New Relic APM
+title: New Relic services
 functional_areas:
   - Cloud
   - Integration
   - Setup
   - Services
 ---
 
-[New Relic for application performance management (APM)](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-new-relic-apm) is a software analytics product that helps you analyze and improve application interactions. Each {{site.data.var.ece}} plan includes a New Relic APM license that supports up to three instances across all environments. 
-
-New Relic APM provides the following features to Magento:
+[New Relic Infrastructure](https://newrelic.com/products/infrastructure) automatically connects with your application data and performance analytics to provide dynamic server monitoring and is standard for {{site.data.var.ece}} accounts. [New Relic for application performance management (APM)](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-new-relic-apm) is a software analytics product that helps you analyze and improve application interactions. New Relic APM provides the following features to Magento:
 
 -  **Focus on specific transactions**—Actively mark and monitor key customer actions in your site, such as adding to the cart, checking out, or processing a payment.
 -  **Database query monitoring**—Locate and monitor database queries affecting performance.
 -  **App Map**—View all application dependencies within your Magento site, extensions, and external services.
 -  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event.
 
-By default, the {{site.data.var.ece}} Pro plan includes New Relic on the Staging and Production environments. If it is not installed, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation. For {{site.data.var.ece}} Pro plan customers also need to submit a support ticket to use New Relic on an Integration branch. 
-
-
 {:.bs-callout .bs-callout-info}
-Your {{site.data.var.ece}} account includes the software for the New Relic APM service along with a license key. You do not need to purchase or install any additional software.
+Your {{site.data.var.ece}} account includes the software for the New Relic service along with a license key. You do not need to purchase or install any additional software. For Pro accounts, if New Relic is not installed on the Staging and Production environments, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation.
```

Comment:
For PRO only APM goes to STG+PROD. Infra goes to PROD only. This needs to be clear here.